File: src/main/java/ru/scarlet/filestorage/dto/UserDto.java
package ru.scarlet.filestorage.dto;

import lombok.Value;

import java.io.Serializable;

/**
 * DTO for {@link ru.scarlet.filestorage.entity.User}
 */
@Value
public class UserDto implements Serializable {
    String login;
    String password;
    String name;
    String lastName;
    String patronymic;
}

File: src/main/java/ru/scarlet/filestorage/dto/UserDto.kt
package ru.scarlet.filestorage.dto

import lombok.Value
import java.io.Serializable

/**
 * DTO for [ru.scarlet.filestorage.entity.User]
 */
@Value
data class UserDto  (
    var login: String? = null,
    var password: String? = null,
    var name: String? = null,
    var lastName: String? = null,
    var patronymic: String? = null
) : Serializable